extension.driver.php

```diff
@@ -86,8 +90,7 @@ public function install()
 
     public function uninstall()
     {
-        if(parent::uninstall() == true)
-        {
+        if (parent::uninstall() == true) {
```

Comment:
> , but I am a bad programmer.

No you're not! And php does makes a difference between the 2 approaches, it some cases those checks are needed.. But using this approach should be banned

> Should I add that to the PR?

Yes please!